examples/web-worker/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>stuff</title>
```

Comment:
You could rephrase it to "web-worker" :P

Reply:
Renamed to `Web Worker` :+1: